src/main/java/org/openstreetmap/atlas/checks/validation/points/LoneNodeCheck.java

```diff
@@ -0,0 +1,112 @@
+package org.openstreetmap.atlas.checks.validation.points;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import org.openstreetmap.atlas.checks.base.BaseCheck;
+import org.openstreetmap.atlas.checks.flag.CheckFlag;
+import org.openstreetmap.atlas.geography.atlas.items.Area;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasObject;
+import org.openstreetmap.atlas.geography.atlas.items.LineItem;
+import org.openstreetmap.atlas.geography.atlas.items.Node;
+import org.openstreetmap.atlas.geography.atlas.items.Point;
+import org.openstreetmap.atlas.tags.HighwayTag;
+import org.openstreetmap.atlas.tags.RailwayTag;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+
+/**
+ * This check verifies lone {@link Node}s with highway tag that they follow the tagging principles.
+ *
+ * @author mm-ciub
+ */
+public class LoneNodeCheck extends BaseCheck<Long>
+{
+
+    private static final long serialVersionUID = -1489101405354234053L;
+
+    private static final List<String> DEFAULT_HIGHWAY_VALUES_CHECKED = List.of("crossing",
+            "turning_circle", "traffic_signals", "stop", "give_way", "motorway_junction",
+            "mini_roundabout", "passing_place", "turning_loop");
+    private static final String LONE_NODE_INSTRUCTION = "This node {0,number,#} has a Highway tag but is not part of any way that has a highway or railway tag. Either add such a tag to the appropriate parent or remove the highway tag from the node.";
+    private static final List<String> FALLBACK_INSTRUCTIONS = Collections
+            .singletonList(LONE_NODE_INSTRUCTION);
+
+    private List<String> highwayValuesToCheck;
+
+    /**
+     * Default constructor.
+     *
+     * @param configuration
+     *            the JSON configuration for this check
+     */
+
+    public LoneNodeCheck(final Configuration configuration)
+    {
+        super(configuration);
+        this.highwayValuesToCheck = this
+                .configurationValue(configuration, "valid.highway.tag",
```

Comment:
Please add this key to the configuration with the default value, and add a description of it to the check's MD file.